### PR TITLE
Email for reviews

### PR DESCRIPTION
**Pull request by surekhag**

### CODE REVIEW

File: server/config/env/dev.js [hunk 1]
```diff
@@ -3,5 +3,16 @@ module.exports = {
   port: 3001, // change with development port
   mongoUrl: "mongodb://localhost:27017/EMS",
   secret: "#$up3r$up3r$3cr3t#@@#",
-  tokenExpiry: "1h"
+  tokenExpiry: "1h",
```
Comment (bhushan-oe) [inline server/config/env/dev.js:6]: Add similar settings in prod.js file

File: server/app/service/ses_client.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+AWS.config.update({
+    accessKeyId: config.aws.key,
+    secretAccessKey: config.aws.secret,
+    region: config.aws.ses.region
+});
+
+const ses = new AWS.SES({apiVersion: '2010-12-01'});
+
+const sendEmail = (to, subject, message) => {
+    const from = 'surekha.gadkari@objectedge.com';
```
Comment (bhushan-oe) [inline server/app/service/ses_client.js]: We need 'from' as parameter to function and default value for from can be read from config

File: server/app/service/ses_client.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        },
+        ReturnPath: from ? from : config.aws.ses.from.default,
+        Source: from ? from : config.aws.ses.from.default,
+    };
+
+    ses.sendEmail(params, (err, data) => {
+        if (err) {       
+            return console.log(err, err.stack);
+        } else {
+            console.log("Email sent.", data);
```
Comment (bhushan-oe) [inline server/app/service/ses_client.js:44]: please remove console. For logging at server side we can use logging package.

File: server/app/api/controllers/peer_review.js [hunk 4]
```diff
@@ -21,11 +24,14 @@ module.exports = {
         },
         function (err, result) {
           if (err) next(err);
-          else
-            res.json({
+          else{
+             const peerName ="surekha.test.email";
+             emailProvider.sendEmail("surekha.gadkari@objectedge.com", 'Peer Review  ', emails.peerReviewEmailTemplate(peerName))
```
Comment (bhushan-oe) [inline server/app/api/controllers/peer_review.js]: Please use variable for hardcoded values.

**Review by bhushan-oe — CHANGES_REQUESTED** [review]

File: server/config/env/dev.js (hunk 1 above)
Comment (surekhag) [reply server/config/env/dev.js:6]: Done.

Comment (surekhag) [reply server/config/env/dev.js:6]: tokenExpiry: "1h"  this is missing in prod .. @bhushan-oe  Is this intentional?

File: server/app/service/ses_client.js (hunk 2 above)
Comment (surekhag) [reply server/app/service/ses_client.js]: The initial implementation that only. Updated.

File: server/app/api/controllers/peer_review.js (hunk 4 above)
Comment (surekhag) [reply server/app/api/controllers/peer_review.js]: Done.

**Review by bhushan-oe — APPROVED** [review]

**Pull request merged**